package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "pnpm": "pnpm",
```

Comment:
pnpm would be installed globally as per the README.md, so there wouldn't be a need to have a separate script named pnpm